### PR TITLE
Cleanup dependencies in package.json

### LINKED ISSUE #2017: Cleanup package.json
Half the time of our build is spent on `npm install`. Let's clean up all unused packages to lower our time

### PR DESCRIPTION
**Pull request by robwalch**
### This PR will...
Remove jsQuery, require.js (plus karma-require) and underscore dependencies.

### Why is this Pull Request needed?
These dependencies are not being used and are not needed.

#### Addresses Issue(s):
Fixes #2017

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -13,19 +13,12 @@ const webpackTestConfig = merge(webpackConfig, {
...
-            root: '$'
-        },
         sinon: 'sinon'
     },
     resolve: {
         alias: {
             'test/underscore': path.resolve(__dirname + '/node_modules/underscore/underscore.js'),
             'utils/video': path.resolve(__dirname + '/test/mock/video.js'),
-            // Tests using jQuery: api-test, jwplayer-selectplayer-test, setup-test
-            jquery: path.resolve(__dirname + '/node_modules/jquery/dist/jquery.js'),
```
Comment (jnatalzia) [inline karma.conf.js]: do we need to update any test pages?

Comment (robwalch) [reply karma.conf.js]: There are only two pages in test and neither use `$`.

File: package.json [hunk 2]
```diff
@@ -43,7 +42,6 @@
     "karma-junit-reporter": "1.2.0",
```
Comment (zetagame) [inline package.json]: Do we use karma-junit-reporter anywhere?

File: package.json [hunk 3]
```diff
@@ -59,13 +57,11 @@
     "postcss-safe-parser": "3.0.1",
```
Comment (zetagame) [inline package.json]: Same question for postcss-safe-parser

File: package.json (hunk 2 above)
Comment (robwalch) [reply package.json]: Jenkins

Comment (robwalch) [reply package.json]: Run ‘yarn why’. Is it a peer dependency?

File: package.json (hunk 3 above)
Comment (robwalch) [reply package.json]: Run ‘yarn why’. Is it a peer dependency?

Comment (robwalch) [reply package.json]:
```
$ yarn why postcss-safe-parser                                                                                                                                                                                                            
 🤔  Why do we have the module "postcss-safe-parser"...?
=> Found "postcss-safe-parser@3.0.1"
info Has been hoisted to "postcss-safe-parser"
info Reasons this module exists
   - Specified in "devDependencies"
   - Hoisted from "stylelint#postcss-safe-parser"
```

**Pull request merged**